source/includes/documentation/_vue_js_elements.md

```diff
@@ -2964,3 +2964,31 @@ Available methods in Java JDI Light:
 
 For examples of usage see: [JDI Vuetify Chip tests](https://github.com/jdi-testing/jdi-light/blob/vuetify-develop/jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/ChipsTests.java).
 
+### 5.39 Aspect Ratios
```

Comment:
Sorry, failed to find this poin "7.	Element code (with reference to framework version - IMPORTANT!)" Could you please, clarify :)